TaylorFramework/Modules/BoundariesKit/Component.swift

```diff
@@ -41,7 +41,7 @@ extension Component : Equatable {}
 func ==(lhs: Component, rhs: Component) -> Bool {
     if lhs.range != rhs.range { return false }
     if lhs.type != rhs.type { return false }
-    if lhs.name !~== lhs.name { return false }
+    if lhs.name != lhs.name { return false }
     if !(Set(lhs.components) == Set(rhs.components)) { return false }
     
     return true
```

Comment:
It doesn't make much difference. The only advantage of this approach is that we have earlier returns which (scaled up) may lead to performance improvements.